regclient/regclient
# Upgrade to Go 1.20 support

Signed-off-by: Brandon Mitchell <git@bmitch.net>



### Fixes issue

Linting fails with latest staticcheck.


### Describe the change

Upgrades to Go 1.20, fixes lint errors.



### How to verify it

Update Go to 1.20 and update linter with: `make -B $(go env GOPATH)/bin/staticcheck`. `make lint` should then work.


### Changelog text

Update Go support to 1.20


### Please verify and check that the pull request fulfills the following requirements
